test/parallel/test-http-early-hints.js

```diff
@@ -99,47 +99,6 @@ const testResBody = 'response content\n';
   }));
 }
 
-{
-  // Happy flow - empty array
-
-  const server = http.createServer(common.mustCall((req, res) => {
```

Comment:
Why did we remove this test?

Reply:
With the removal of the requirement to send a link header this test seemed identical to the empty object test. Could be put back if you disagree.